coffee-chats/src/main/webapp/src/requests/tags.ts

```diff
@@ -0,0 +1,13 @@
+const tagsURL = '/api/tags';
+
+export const fetchTags = () => {
```

Comment:
Is this different from `export function fetchTags()`?

Reply:
It is the same, but a different syntax. Arrow functions are preferred for their simplicity over declaring a function explicitly (see [styleguide link](https://google.github.io/styleguide/jsguide.html#features-functions-arrow-functions))

Reply:
Hm, I was assuming it's preferred only when capturing `this`. Sure